client/src/pages/JoinGame.tsx

```diff
@@ -11,6 +13,15 @@ const JoinGame = () => {
     socket.emit("JOIN_GAME", gameCode, playerName);
```

Comment:
Should this be in a useEffect too? Also don't we need game code from the URL params?

Reply:
This is sending the JOIN_GAME even to the server, so we need to input the game code that a Player is trying to join too.